-------------------------------------------------------------------------------
Test set: master.sedemo.CustomerTests_C4.test.datamodel.Customer_400_ContactsTest
-------------------------------------------------------------------------------
Tests run: 8, Failures: 4, Errors: 0, Skipped: 0, Time elapsed: 0.001 s <<< FAILURE! - in master.sedemo.CustomerTests_C4.test.datamodel.Customer_400_ContactsTest
test401_addContactsCornerCases  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: array contents differ at index [0], expected: < eric@gmail.com  > but was: <eric@gmail.com>
	at master.sedemo.CustomerTests_C4.test.datamodel.Customer_400_ContactsTest.test401_addContactsCornerCases(Customer_400_ContactsTest.java:54)

test402_addContactsCornerCases  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: array contents differ at index [0], expected: <"eric@gmail.com"> but was: <eric@gmail.com>
	at master.sedemo.CustomerTests_C4.test.datamodel.Customer_400_ContactsTest.test402_addContactsCornerCases(Customer_400_ContactsTest.java:69)

test403_addContactsMinimumLength  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected java.lang.IllegalArgumentException to be thrown, but nothing was thrown.
	at master.sedemo.CustomerTests_C4.test.datamodel.Customer_400_ContactsTest.test403_addContactsMinimumLength(Customer_400_ContactsTest.java:90)

test410_deleteContactRegularCases  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <3> but was: <2>
	at master.sedemo.CustomerTests_C4.test.datamodel.Customer_400_ContactsTest.test410_deleteContactRegularCases(Customer_400_ContactsTest.java:141)

